docs/proposals/automated-node-removal.md

```diff
@@ -0,0 +1,60 @@
+# Proposal: Automation of node removal procedure
+
+Last updated: 24.06.2021
+
+## Abstract
+
+Implementation of the option to delete CSI CRs from a node, which going to be removed. 
+
+## Background
+
+User manual actions for node removal are described in [doc](https://github.com/dell/csi-baremetal/blob/master/docs/node-removal.md). The goal of this POC is automating most of the steps of that procedure. 
+
+## Proposal
+
+CSI CRs deletion(Volumes, AC, ACR, Drives, LVGs) will be triggerred on deletion of appropriate csibmnode and we be performed by node-controller.
```

Comment:
I recommend to use node annotation as an API of node removal. Please put some notes about node removal and replacement difference.

Reply:
updated